README.md

```diff
@@ -1 +1,11 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# My name is Alia Ai-Ming Tehzeeb Shahzad. 
+
+I'm a **third year** at the College.
+
+I'm double-majoring in *public policy* and *creative writing* inshallah. 
+
+[This](https://www.youtube.com/watch?v=7D3Ud2JIFhA) is my favorite album. 
```

Comment:
Sometimes it's helpful to be a little more specific about where a link is going in the text that contains the link (but this is more a stylistic choice, the formatting is fine)